rust/core-lib/src/tabs.rs

```diff
@@ -319,6 +319,7 @@ impl CoreState {
             // handled at the top level
             ClientStarted { .. } => (),
             SetLanguage { view_id, language_id } => self.do_set_language(view_id, language_id),
+            #[cfg(feature = "notify")]
```

Comment:
Making a check here since fuchsia does not have notify.

Reply:
So I think the best way to handle this is to remove the `cfg` here and above, and instead have two versions of `do_toggle_tail`; one of them is exactly what you have right now, with `#[cfg(feature = "notify")]`, and the other one looks like,

```rust
 #[cfg(not(feature = "notify"))]
fn do_toggle_tail(&mut self, _view_id: ViewId, _enabled: bool) {
    warn!("do_toggle_tail called without notify feature enabled.");
}
```

You can see another example of this pattern (without the `warn!` macro) here: https://github.com/xi-editor/xi-editor/blob/master/rust/core-lib/src/tabs.rs#L671